RoslynSecurityGuard/RoslynSecurityGuard.Test/Tests/CsrfTokenAnalyzerTest.cs

```diff
@@ -26,23 +26,23 @@ protected override IEnumerable<MetadataReference> GetAdditionnalReferences()
         public void CsrfDetectMissingToken()
         {
             var test = @"
-using System;
-using System.Diagnostics;
-using System.Web.Mvc;
+                using System;
```

Comment:
Code samples are blobs of text. They don't need to be aligned with the test code.
But either way, I don't have a preference.